string class_id
int64 instance_id
---
ObjectPose pose_query_result